src/base/ArbitraryBase.ts

```diff
@@ -3,18 +3,22 @@
  *
  * @since 1.0.0
  */
-import * as fc from 'fast-check'
+import type * as FastCheck from 'fast-check'
 import { identity, pipe } from 'fp-ts/function'
 import * as RA from 'fp-ts/ReadonlyArray'
 import * as RR from 'fp-ts/ReadonlyRecord'
-import * as RTup from 'fp-ts/ReadonlyTuple'
 import * as S from 'io-ts/Schemable'
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Any = any
+
 /**
  * @since 1.0.0
  * @category Model
  */
-export interface Arbitrary<A> extends fc.Arbitrary<A> {}
+export interface Arbitrary<A> {
+  readonly arbitrary: (fc: typeof FastCheck) => FastCheck.Arbitrary<A>
+}
```

Comment:
I would actually prefer not to have the object wrapper (because it makes it a gigantic pain in the butt).  But it takes this shape in `fp-ts/codec` 

https://github.com/fp-ts/codec/blob/a0dce3915cfe00dd78c2f27b983df93fc528588d/src/Arbitrary.ts#L24

Maybe it's not worth the extra effort, but the work's already done 🤷‍♂️